### PR TITLE
Fix makefile and add missing ftp structure

### PR DESCRIPTION
**Pull request by gemmai95**

### CODE REVIEW

**Review by JamesGardiner — CHANGES_REQUESTED** [review]
Could you update the changelog please?

File: Makefile [hunk 1]
```diff
@@ -40,6 +40,6 @@ start:
 build: check-env clone
 	@ printf "\n[${GREEN} Generating environment variables... ${NO_COLOR}]\n"
 	cd ${SDX_HOME}/sdx-common && pip install .; cd -
-	cd ${SDX_HOME}/sdx-ops && ${PYTHON3} sdx/ops/configure.py --env > ${SDX_HOME}/sdx-compose/env/private.env ;cd -
+	cd ${SDX_HOME}/sdx-ops && python sdx/ops/configure.py --env > ${SDX_HOME}/sdx-compose/env/private.env ;cd -
```
Comment (necrophonic) [inline Makefile:43]: Why changing from the parameterised call? The local installed python3 may not be available as a specific `python` command in the path

Comment (JamesGardiner) [reply Makefile:43]: This repo states that users should have an activated virtual environment before running, that has been setup to use Python3.

Comment (JamesGardiner) [reply Makefile:43]: But, to be honest, I don't even know why we are generating keys. Don't we have test keys in compose?

Comment (necrophonic) [reply Makefile:43]: It's inconsistent with the rest of the makefile though that still uses `${PYTHON3}`

Comment (JamesGardiner) [reply Makefile:43]: That should be taken out. README also needs updating as it still references PYTHON3 variable @gemmai95

Comment (warrenbailey) [reply Makefile:43]: I actually find it really annoy you need a virtual environment to run a docker-compose script, it shouldn't even be using Python, all it's doing is building and starting docker containers.

I agree with James, why are we even generating keys? Why not just use the test keys? We could remove all that and keep the script much simpler.

Though if we're determined to create keys just convert the configure.py script to a bash script, it's basically 4 openssl commands. Then we don't have to create a virtual environments to install sdx-common and we probably don't even need Python on the path at all.  

Either of these options simplifies a Makefile that at the minute doesn't work out of the box.

Comment (necrophonic) [reply Makefile:43]: The bash script is probably the way to go as Warran suggests 👍 

I would just check keys in, but even if they're technically fake and not used for anything there's always a risk one day they _may_ actually get used, plus any security scans we do of our repos would probably be false-positiving by picking them up.

**Review by necrophonic — DISMISSED** [review]
Ok to go as - is. Will raise a tech debt card to create a bash script for the keys as mentioned in the review.

**Review by JamesGardiner — APPROVED** [review]

**Pull request merged**